src/main/resources/feedbackQuestionTextSubmissionFormTemplate.html

```diff
@@ -1,7 +1,9 @@
 <div class="panel panel-default panel-body"
-    ${disabled} id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
-    data-length-text-id="responseLength-${questionIndex}-${responseIndex}" data-recommended-text="${recommendedLength}">
+     data-is-session-closed="${isSessionClosed}"
+     id="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     name="${Const.ParamsNames.FEEDBACK_RESPONSE_TEXT}-${questionIndex}-${responseIndex}"
+     data-length-text-id="responseLength-${questionIndex}-${responseIndex}"
+     data-recommended-text="${recommendedLength}">
```

Comment:
@amarlearning Apoloogies for being unclear.I meant that you should revert back to the orginal alignment of attributes (from the TEAMMATES/master branch) and add your one data-attribute on one of the existing lines. See the diff I posted in https://github.com/TEAMMATES/teammates/pull/7811#discussion_r129121116.